Control coverage/Feature/AppService.md

```diff
@@ -566,3 +568,51 @@ Managed Service Identity (MSI) allows your app to easily access other AAD-protec
 
 ___ 
 
+## Azure_AppService_DP_Use_Secure_FTP_Deployment
+
+### Display Name 
+App Services should use secure FTP deployments.
+
+### Rationale 
+FTPS is used to enhance security for your Azure Web Application as it adds an extra layer of security to the FTP protocol,Enforcing FTPS-only Access for your Azure App Services apps can guarantee that the encrypted traffic between the web apps and servers and the FTP clients cannot be decrypted by malicious actors.
+
+### Control Spec 
+
+ >**Passed :**
+ > - FTP State found to be configured to FTPS (secure FTP) or marked as disabled for all slots in the App Service.
+ >
+> **Failed :**
+> - FTP State should either be configured to FTPS (secure FTP) or marked as disabled for all slots in the App Service.
+>
+> **Note :** If no Microsoft Defender for Cloud (MDC) assessment is found for the App Service, response from the ARM API is considered for the evaluation.
+>
+
+### Recommendation 
+- **Azure Portal** 
+
+	 To make production slot compliant:
+	 Go to Azure Portal --> your App Service --> Settings --> Configuration --> General Settings --> FTP state -->(Choose FTPS Only/Disabled based on requirement)--> Save.
+
+	 To make non-production slot compliant
+	 Go to Azure Portal --> your App Service --> Deployment --> Deployment slots --> Select slot --> Settings --> Configuration --> General Settings --> FTP state -->(Choose FTPS Only/Disabled based on requirement)--> Save.
+
+
+### Azure Policy or ARM API used for evaluation 
+
+- Azure Policy (built-in):
+  [Function apps should require FTPS only](https://ms.portal.azure.com/#view/Microsoft_Azure_Policy/PolicyDetailBlade/definitionId/%2Fproviders%2FMicrosoft.Authorization%2FpolicyDefinitions%2F399b2637-a50f-4f95-96f8-3a145476eb15)
+  <br />
+- Azure Policy (built-in):
+  [App Service apps should require FTPS only](https://ms.portal.azure.com/#view/Microsoft_Azure_Policy/PolicyDetailBlade/definitionId/%2Fproviders%2FMicrosoft.Authorization%2FpolicyDefinitions%2F4d24b6d4-5e53-4a4f-a7f4-618fa573ee4b)
```

Comment:
Why 2 times?

Reply:
If it is a different definition, Call out the difference clearly

Reply:
Resolving as these both are different MDC assessments.